test/unit/presenters/publishing_api_presenters/classification_test.rb

```diff
@@ -0,0 +1,34 @@
+require 'test_helper'
+
+class PublishingApiPresenters::ClassificationTest < ActiveSupport::TestCase
+  test "presents a valid placeholder 'topic' content item" do
+    topic = create(:topic, name: "Defence and armed forces")
+    public_path = "/government/topics/defence-and-armed-forces"
+
+    expected = {
+      base_path: public_path,
+      publishing_app: "whitehall",
+      rendering_app: "whitehall-frontend",
+      format: "placeholder_topic",
```

Comment:
I think we should use a `format` of `policy_area`/`placeholder_policy_area` to be consistent with the label we use on [GOV.UK](https://www.gov.uk/government/topics/public-health). After talking with @rboulton, we also settled on `policy_areas` for the key in the links hash which will contain these things. See: https://github.com/alphagov/govuk-content-schemas/blob/master/formats/statistics_announcement/publisher/links.json#L13